wakuv2/waku.go

```diff
@@ -1340,7 +1189,29 @@ func (w *Waku) Start() error {
 
 	go w.sendQueue.Start(w.ctx)
 
-	go w.checkIfMessagesStored()
+	if w.cfg.EnableStoreConfirmationForMessagesSent {
```

Comment:
Maybe move everything under the if condition to a separate function? 
That way Start function would look leaner.